deploy/docs/Non_Helm_Installation.md

```diff
@@ -60,273 +39,153 @@ First, you'll need to set up the relevant [fields](https://help.sumologic.com/Ma
-sed 's/\$NAMESPACE'"/<NAMESPACE>/g" | \
-sed 's/cluster kubernetes/cluster <CLUSTER_NAME>/g' | \
-kubectl -n sumologic apply -f -
-```
-
-### Customize configuration
-
-If you need to customize the configuration there are two commands to run. First, get the `fluentd-sumologic.yaml` manifest with following command:
-
-```sh
-curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.0/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/sumologic/g" >> fluentd-sumologic.yaml
-```
+The following parameter is optional, but we recommend setting it.
+* __sumologic.clusterName__ - An identifier for your Kubernetes cluster.  This is the name you will see for the cluster in Sumo Logic. Default is `kubernetes`.
 
-Next, customize the provided YAML file. Our [plugin](../../fluent-plugin-events/README.md#fluent-plugin-events) allows you to configure fields for events. Once done run the following command to apply the `fluentd-sumologic.yaml` manifest.
+First, you will need to apply the required CRD's for the Prometheus Operator. This is required before generating the YAML.
 
-```sh
-kubectl -n sumologic apply -f fluentd-sumologic.yaml
+```bash
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
 ```
 
-The manifest will create the Kubernetes resources required by Fluentd.
-
-
-### Verify the pods are running
+Next you will generate the YAML to apply to your cluster.  The following command contains the minimum parameters that can generate the YAML to setup Sumo Logic's Kubernetes collection. This command will generate the YAML and pipe it a file called `sumologic.yaml`.
 
-```sh
-kubectl -n sumologic get pod
+```bash
+kubectl run tools \
+  -it --quiet --rm \
+  --restart=Never \
+  --image sumologic/kubernetes-tools:1.0.0-rc.0 -- \
+  template \
+  --name-template 'collection' \
+  --set sumologic.accessId='<ACCESS_KEY>' \
+  --set sumologic.accessKey='<ACCESS_ID>' \
+  --set sumologic.clusterName='<CLUSTER_NAME>' \
+  | tee sumologic.yaml
```

Comment:
Does it work if we do not set namespace explicitly?

Reply:
Hrmm, before we made all the changes I thought it did as I did test w/o.  Now it is giving me this error `Error: cannot set --name-template and also specify a name`.  So it 100% does not work w/o the namespace then correct?

Reply:
Ok so https://github.com/SumoLogic/sumologic-kubernetes-tools/pull/25 will fix the error but yes, it does work when a namespace is not specified. I ran without the namespace flag and simply ran kubectl apply in the namespace i was currently configured in. It does not work when you try to apply the yaml with the --namespace flag because there are some objects generated that have to be installed in kube-system and already have that set as part of the templates.  Thats why i have the step to change your context.